### ISSUE TITLE
Adjust 2FA Guide Styling

### ISSUE THREAD

**Issue by daniellex0** [hackforla/website#1332]
### Overview
We need to adjust the 2FA guide styling to make it more legible / user-friendly. This is a temporary fix while the Guide pages are going through an overhaul.

### Action Items
- [x] Verify whether changing the styling on the [2FA guide](https://www.hackforla.org/guide-pages/2FA) will affect other guides' styling (for ex., the [survey reporting guide](https://www.hackforla.org/guide-pages/survey-reporting-dashboard-guide))
   - [x] If it will affect other guides, alert @daniellex0 by commenting on this issue and double check the Survey Reporting guide when continuing to the following step to make sure it still looks good as well with the changes
- [ ] Change the ".section-container" (page card) margin to **2% 12%** (instead of 5%)

### Resources/Instructions
[2FA guide
](https://www.hackforla.org/guide-pages/2FA)[Survey Reporting guide
](https://www.hackforla.org/guide-pages/survey-reporting-dashboard-guide)[HfLA Site Architecture ](https://github.com/hackforla/website/wiki/Hack-for-LA's-Site-Architecture)

**Comment by RobertaFricker** [comment]
@daniellex0 
Hi Danielle, I reviewed all 10  _guide-pages to confirm whether they're using the section-container class:

![image](https://user-images.githubusercontent.com/81199122/114761925-fe39b180-9d15-11eb-9fa2-0cbf514f0667.png)

The following 3 guide-pages are using it and here are the links:
**2FA**                
https://www.hackforla.org/guide-pages/2FA 
**github-issues**
https://www.hackforla.org/guide-pages/github-issues
**survey reporting dashboard**
https://www.hackforla.org/guide-pages/survey-reporting-dashboard-guide.html

Please advise on whether you'd like me to change the margins in the section-container class:
![image](https://user-images.githubusercontent.com/81199122/114763074-44434500-9d17-11eb-9879-59764936f012.png)

Thanks,

**Comment by RobertaFricker** [comment]
@daniellex0 
@jbubar 
After reviewing the 3 guide-pages:  2FA, github-issues and survey-reporting-dashboard, changing the section-container margin doesn't adversely affect the page display and resizing.  However the 2FA.html page has two directives for .accordion margins.  One is pointing to getting-started-page.scss (see screenshot below) and the other 2 are pointing to the 2FA.scss file (see screenshot below).

![image](https://user-images.githubusercontent.com/81199122/114789334-cfccce00-9d37-11eb-9b52-4c94daddd93e.png)

![image](https://user-images.githubusercontent.com/81199122/114789444-fd197c00-9d37-11eb-83fc-55ab556ef637.png)


For the 2FA guide-page, the ipad landscape display becomes corrupted with the lines connected to the arrows run off the right side of the page:
![image](https://user-images.githubusercontent.com/81199122/114789276-b3309600-9d37-11eb-999f-38d76c590384.png)

In conclusion, there is an issue with the .accordion class that's unrelated to the .section-container change requested in this issue.

**Comment by daniellex0** [comment]
Great, I'm glad changing the margin doesn't affect the 3 guides- in that case the changes should be good to go/ ready for a PR!

Regarding the min-width of the accordion- I'm not sure I fully understand the problem with having min-width. I see that it's min-width 150px in the smaller screen size breakpoint and min-width 835px in the larger screen size. When min-width is disabled, the accordion gets messed up. So it seems to be working well... Is the issue that only in iPad landscape it gets messed up? Does that mean there should be a different breakpoint set? If so, perhaps a new issue can be created for that.

**Comment by erikaBell** [comment]
Left comment on pull request, regarding `min-width` comments. #1406

**Comment by averdin2** [comment]
@daniellex0 I was looking into this issue a bit regarding the min-width, and what seems to be the issue is having min-width be 835px. When in an iPad landscape view, it causes the accordion to extend past the card as you can see in @RobertaFricker's image. I still need to look into that issue a little more to see if having min-width is necessary, but it actually seems to be the case that when min-width is disabled, it fixes the accordion. I agree that this issue is probably good to go, and the accordion problem can be a new issue.

**Comment by daniellex0** [comment]
@averdin2 Hm yeah, I just commented on the PR I'm realizing you might be right... although I do wonder what the purpose of the min-width was if anything.. maybe you can ask the devs tomorrow in the breakout room (if there's time, this isn't time sensitive) if anyone has any idea..?
